Taskfile.yml

```diff
@@ -182,16 +175,30 @@ tasks:
     - CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags "-X {{.CONTROLLER_PACKAGE}}/pkg/version.BuildVersion={{.VERSION}}" -o ./bin/{{.CONTROLLER_APP}}
 
   controller:docker-build:
-    desc: Generated the {{.CONTROLLER_APP}} Docker file.
+    desc: Builds the {{.CONTROLLER_APP}} Docker image.
     dir: "{{.CONTROLLER_ROOT}}"
     deps: [controller:build]
     sources:
     - Dockerfile
     - ./bin/{{.CONTROLLER_APP}}
     cmds:
     - docker build . -t {{.CONTROLLER_DOCKER_IMAGE}}
+
+  controller:docker-build-and-save:
+    desc: Generates the {{.CONTROLLER_APP}} Docker file.
+    dir: "{{.CONTROLLER_ROOT}}"
+    deps: [controller:docker-build]
+    cmds:
     - docker save {{.CONTROLLER_DOCKER_IMAGE}} > bin/{{.CONTROLLER_DOCKER_IMAGE}}.tar
 
+  controller:docker-push-remote:
```

Comment:
minor: Maybe fix my awkwardly named `controller:docker-push-local-reg` to `controller:docker-push-local` to match this?